boards/ESP8266_BOARD.py

```diff
@@ -46,8 +46,8 @@
   'saved_code' : {
     'address' : 0x78000,
     'page_size' : 4096,
-    'pages' : 3,
-    'flash_available' : 468, # firmware can be up to this size
+    'pages' : 4,
+    'flash_available' : 472, # firmware can be up to this size
```

Comment:
I understand the code size has increased by a page, however don't understand why the firmware can be larger?

Reply:
Gave up one Wifi page and one flash, for details see #1380

Reply:
> espruino_esp8266_user1.bin, max 118 pages  = max size 472 KB

Where did the one for flash come from? Isn't the phy data stored in the last block by the operating system?